src/main/java/org/eclipse/xpanse/terraform/boot/models/response/TerraformResult.java

```diff
@@ -16,11 +17,12 @@
 @Builder
 public class TerraformResult {
 
+    @Schema(description = "Value of destroy scenario filled into the result of destroy request. "
+            + "Valid values: destroy,rollback,purge.")
+    private String destroyScenario;
```

Comment:
we must use the same DestroyScenario enum here as well.

Reply:
Updated.